.importlinter

```diff
@@ -0,0 +1,33 @@
+[importlinter]
+root_package=mlrun
+include_external_packages=True
+
+
+[importlinter:contract:1]
+name=common modules shouldn't import other mlrun utilities
+type=forbidden
+source_modules=
+    mlrun.common
+
+forbidden_modules=
+    mlrun.api
+    mlrun.artifacts
+    mlrun.data_types
```

Comment:
can do `mlrun.*` instead? or something regex-like? to avoid forgetting adding packages here?

Reply:
I didn't find such option